services/monitors/monitors/parsers.go

```diff
@@ -14,41 +14,25 @@ func (monitor *Monitor) ToMap() map[string]any {
 	monitorMap["name"] = monitor.Name
 	monitorMap["description"] = monitor.Description
 	monitorMap["status"] = monitor.Status
-	monitorMap["polling_frequency_in_minutes"] = monitor.PollingFrequencyInMinutes
 	monitorMap["metric_id"] = monitor.MetricId
 	monitorMap["node_id"] = monitor.NodeId
-	monitorMap["statistics"] = []any{monitor.Statistics.ToMap()}
-	monitorMap["expression"] = []any{monitor.Expression.ToMap()}
+	monitorMap["rule"] = []any{monitor.Rule.ToMap()}
 	monitorMap["action_templates"] = helper.ParseToMaps(monitor.ActionTemplates)
 	monitorMap["action_template_ids"] = monitor.ActionTemplateIds
 	monitorMap["tags"] = helper.ParseToMaps(monitor.Tags)
 	monitorMap["created_at"] = monitor.CreatedAt
 	monitorMap["created_by"] = monitor.CreatedBy
 	monitorMap["last_modified_at"] = monitor.LastModifiedAt
 	monitorMap["last_modified_by"] = monitor.LastModifiedBy
+	monitorMap["type"] = "REALTIME"
 	return monitorMap
 }
 
-func (statistics *Statistics) ToMap() map[string]any {
-	statisticsMap := make(map[string]any)
-	statisticsMap["last_evaluation"] = []any{statistics.LastEvaluation.ToMap()}
-	return statisticsMap
-}
-
-func (evaluation *Evaluation) ToMap() map[string]any {
-	evaluationMap := make(map[string]any)
-	evaluationMap["timestamp"] = evaluation.Timestamp
-	evaluationMap["value"] = evaluation.Value
-	evaluationMap["status"] = evaluation.Status
-	return evaluationMap
-}
-
-func (expression *Expression) ToMap() map[string]any {
+func (rule *Rule) ToMap() map[string]any {
 	expressionMap := make(map[string]any)
```

Comment:
Can you replace `expressionMap` by `ruleMap`?
```suggestion
	ruleMap := make(map[string]any)
```